lib/game.ex

```diff
@@ -5,28 +5,29 @@ defmodule Game do
             winner: nil, 
             board: %Board{}.spaces
             
-  # convert to private functions?
-  def check_three([x, x, x]), do: true
-  def check_three([a, b, c]), do: false
+  defp check_three([x, x, x]), do: x
+  defp check_three([_a, _b, _c]), do: false
```

Comment:
This function and the one above it - do they give different return values?

Reply:
same values, it just kept giving warnings about not using the variables

Reply:
It looks like the first one will only return if all three values are the same, which is a clever way of checking for a match. I would stick to one return type, however. Either `true | false` or `<winner> | nil`.